iree_tests/pytorch/models/sd-unet-tank/test_cases.json

```diff
@@ -0,0 +1,27 @@
+{
+  "test_cases": [
+    {
+      "name": "splats",
+      "runtime_flagfile": "splat_data_flags.txt",
+      "remote_file_groups": []
+    },
+    {
+      "name": "real_weights",
+      "runtime_flagfile": "real_weights_data_flags.txt",
```

Comment:
These tests timed out (and then were retried... which then ignored the timeout... that's an odd interaction, pytest plugins :P)

https://github.com/nod-ai/SHARK-TestSuite/actions/runs/8433834658/job/23095725951?pr=126#step:10:26

How about increasing the timeout here to 600 seconds? https://github.com/nod-ai/SHARK-TestSuite/blob/73dcb4239cea621e4d5366b04f2bde7ffccd4470/.github/workflows/test_iree.yml#L75-L78 (that's what I have it set to in https://github.com/openxla/iree/pull/16801 over in the IREE repo)